libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloResponse.kt

```diff
@@ -10,12 +10,18 @@ import kotlin.jvm.JvmField
 import kotlin.jvm.JvmName
 
 /**
- * Represents a GraphQL response. GraphQL responses can be partial responses, so it is valid to have both data != null and exception != null
+ * Represents a GraphQL response.
```

Comment:
In retrospect, `ApolloResponse` could have been `ApolloResult` or so. I think this ship has sailed but we can be more precise about what `ApolloResponse` is here:

```suggestion
 * Represents a GraphQL response or an exception if no GraphQL response is received.
```